packages/agents-manager/src/components/agent-dock/index.tsx

```diff
@@ -35,6 +35,11 @@ type NavigationContinuationHook = ( props: {
 	agentId: string;
 } ) => void;
 
+/**
+ * Abilities setup hook type - registers hook-dependent abilities
+ */
+type AbilitiesSetupHook = () => void;
```

Comment:
We could remove the type from /utils/load-external-providers.ts.

By the way, I’d appreciate it if we could reuse the [NavigationContinuationHook](https://github.com/Automattic/wp-calypso/blob/614732877400b178126560671f9aadab32e9275e/packages/agents-manager/src/components/agent-dock/index.tsx#L31-L36) from that file as well.